docs/source/topics/cache.rst

```diff
@@ -0,0 +1,61 @@
+================
+Database caching
+================
+
+.. class:: django_mongodb_backend.cache.MongoDBCache
+
+You can configure :doc:`Django's caching API <django:topics/cache>` to store
+its data in MongoDB.
+
+To use a database collection as your cache backend:
+
+* Set :setting:`BACKEND <CACHES-BACKEND>` to
+  ``django_mongodb_backend.cache.MongoDBCache``
+
+* Set :setting:`LOCATION <CACHES-LOCATION>` to ``collection_name``, the name of
+  the MongoDB collection. This name can be whatever you want, as long as it's a
+  valid collection name that's not already being used in your database.
+
+In this example, the cache collection's name is ``my_cache_collection``::
+
+    CACHES = {
+        "default": {
+            "BACKEND": "django_mongodb_backend.cache.MongoDBCache",
+            "LOCATION": "my_cache_collection",
+        },
+    }
+
+Unlike Django's built-in database cache backend, this backend supports
+automatic culling of expired entries at the database level.
+
+In addition, the cache is culled based on ``CULL_FREQUENCY`` when  ``add()``
+or ``set()`` is called, if ``MAX_ENTRIES`` is exceeded. See
+:ref:`django:cache_arguments` for an explanation of these two options.
```

Comment:
These two paragraphs replaced this one which was copied from the Django docs:

> Unlike other cache backends, the database cache does not support automatic
culling of expired entries at the database level. Instead, expired cache
entries are culled each time ``add()``, ``set()``, or ``touch()`` is called.